example/ios/Podfile

```diff
@@ -0,0 +1,41 @@
+# Uncomment this line to define a global platform for your project
```

Comment:
Dear @sandeshgyw can you please provide some information on what that codes here is based?